src/System.Private.CoreLib/shared/System/Lazy.cs

```diff
@@ -440,9 +440,11 @@ private T CreateValue()
         /// <exception cref="T:System.NullReferenceException">
         /// The <see cref="Value"/> is null.
         /// </exception>
-        public override string ToString()
+        public override string? ToString()
         {
-            return IsValueCreated ? Value.ToString() : SR.Lazy_ToString_ValueNotCreated;
+            return IsValueCreated ?
+                Value!.ToString() : // TODO-NULLABLE: Documented to throw NullReferenceException
```

Comment:
TODO because you want to replace it with ANE (and update docs)?

Reply:
Maybe. I wanted to flag it to revisit at least.